website/templates/home.mako

```diff
@@ -45,6 +45,41 @@
         window.contextVars = $.extend(true, {}, window.contextVars, {
             dashboardInstitutions: ${ dashboard_institutions | sjson, n},
         });
+        // Creating global waffle object to give access to flags, switches and samples
```

Comment:
Rather than writing this code here, could we instead request the JS file, as documented here: https://waffle.readthedocs.io/en/latest/usage/javascript.html ?

The only difference is that instead of using `<script src="{% url 'wafflejs' %}"></script>`, we would need to input the url ourselves. We could put the URL in the global mako context, i.e. in `website.routes.get_globals` so that it can be included in a script tag. Something like

```python
# website/routes
def get_globals():
    #...
    return {
         # ...
          'wafflejs_url': reverse('wafflejs')
    }
```


```mako
## base.mako

<script src="${wafflejs_url}"></script>

Reply:
Getting this wafflejs file in the flask app is the trickiest part.  I spent a long time trying to include it back in September, got stuck, and manually included it myself.

Can you tell me what this `reverse` piece is doing?   I'm not sure where to include the waffle.urls.  They are django url's, how is the flask app supposed to talk to them?